jazz_create-serverless-service/index.js

```diff
@@ -31,17 +31,15 @@ const crud = require("./components/crud")(); //Import the utils module.
     @author:
     @version: 1.0
 **/
-
-module.exports.handler = (event, context, cb) => {
+var user_id;
```

Comment:
You have changed the scope of user_id, serviceId, serviceDataObject, is that intentional?